code/__DEFINES/atmospherics.dm

```diff
@@ -62,7 +62,10 @@
 //GASES
 #define MIN_TOXIC_GAS_DAMAGE				1
 #define MAX_TOXIC_GAS_DAMAGE				10
-#define MOLES_GAS_VISIBLE					0.5		//Moles in a standard cell after which gases are visible
+#define MOLES_GAS_VISIBLE					0.25	//Moles in a standard cell after which gases are visible

```

Comment:
please revert this and other whitespace changes in the file

Reply:
Gases below 0.5 have a really low alpha (13 out of 255). By increasing `MOLES_GAS_VISIBLE`, either the gas "clouds" borders won't blend nicely (as there will be no gases with really low alpha) or gases in not-so-high amounts will be too translucent (as the really low alpha will apply to higher amounts of gas).

Reply:
@nicbn he's not talking about the number change, he's talking about the change in number of spaces there are on line 65

Reply:
yeah sorry perhaps I was unclear I was referring only to the whitespace change on this line (and the ones above it)

Reply:
it's fine tho the other ones were more important